Author: Paul Wise <pabs@debian.org>
Description: Default ShowWebsite so webpage not called on exit
Bug-Debian: http://bugs.debian.org/481998
Last-Update: 2012-03-04
Forwarded: no
--- a/lib/Games/PangZero/Globals.pm
+++ b/lib/Games/PangZero/Globals.pm
@@ -242,6 +242,6 @@
 $Games::PangZero::FullScreen             = 1;
 $Games::PangZero::UnicodeMode            = 0;
 $Games::PangZero::Slippery               = 0;
-$Games::PangZero::ShowWebsite            = 0;
+$Games::PangZero::ShowWebsite            = 1.4.1;
 
 1;
